app/sidekiq/lighthouse/poll_form526_pdf.rb

```diff
@@ -96,6 +96,18 @@ def perform(submission_id) # rubocop:disable Metrics/MethodLength
           Rails.logger.info('PDF found')
           return
         else
+          # Check the submission.created_at date, if it's more than 2 days old
+          # update the job status to pdf_not_found immediately and exit the job
+          unless submission.created_at.between?(Date.current - 2, Date.current)
+            form_job_status.update(
+              status: Form526JobStatus::STATUS[:pdf_not_found],
+              bgjob_errors: bgjob_errors.merge(new_error)
+            )
+            ::Rails.logger.warn(
+              'Submission creation date is over 2 days old. Exiting...'
```

Comment:
Can you make this message more specific so when you see it amongst all the other warnings you know it's from form 526?

Reply:
Absolutely!

Reply:
Thanks! Updates look good!